.github/workflows/ci.yml

```diff
@@ -37,3 +38,25 @@ jobs:
     - name: Build host tooling
       run: |
         cargo build
+
+  # Refs: https://github.com/rust-lang/crater/blob/9ab6f9697c901c4a44025cf0a39b73ad5b37d198/.github/workflows/bors.yml#L125-L149
+  # bors.tech integration
+  ci-success:
+    name: ci
+    if: ${{ success() }}
+    needs:
+      - test
+    runs-on: ubuntu-20.04
+    steps:
+      - name: CI succeeded
+        run: exit 0
+
+  ci-failure:
+    name: ci
+    if: ${{ !success() }}
+    needs:
+      - test
+    runs-on: ubuntu-20.04
+    steps:
+      - name: CI failed
+        run: exit 1
```

Comment:
Hmm, why are these needed? Can't you add all required workflows to bors.toml instead? `cortex-m` doesn't have this, for example.

Reply:
unsure; copied them from a working GHA+bors configuration (heapless)